src/audio/asrc/asrc.c

```diff
@@ -50,16 +55,23 @@ static const struct comp_driver comp_asrc;
 
 /* c8ec72f6-8526-4faf-9d39-a23d0b541de2 */
 DECLARE_SOF_RT_UUID("asrc", asrc_uuid, 0xc8ec72f6, 0x8526, 0x4faf,
-		 0x9d, 0x39, 0xa2, 0x3d, 0x0b, 0x54, 0x1d, 0xe2);
+		    0x9d, 0x39, 0xa2, 0x3d, 0x0b, 0x54, 0x1d, 0xe2);
```

Comment:
@softwarecki sorry - change from the previous comment, we need to keep both UUIDs until the switchover at MTL. This is to ease the topology usage for distros using Intel platforms.